CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualManager.java

```diff
@@ -297,11 +357,8 @@ public final void addMultiChangeOperation(final List<VisualChange> changes) {
         addOperation(constructMultiChangeOperation(changes));
     }
 
-    void signifyProcessorIdle() {
-        addOperation(SIGNIFY_PROCESSOR_IDLE_OPERATION);
-    }
 
-    private final VisualOperation SIGNIFY_PROCESSOR_IDLE_OPERATION = new VisualOperation() {
+    protected static final VisualOperation SIGNIFY_PROCESSOR_IDLE_OPERATION = new VisualOperation() {
```

Comment:
So this is a dangerous change I feel. I think there is one visual manager per graph which "managers" the interactions for said graph. Making this static would mean that all of these are shared for all graphs which may not be what you want. Requires significant testing if so.

@aldebaran30701 was this fixing a bug or something else like making it easier to test?

Reply:
I assume the values contained within the VisualOperation wouldn't change, and that making them static would just allow only one to be created for all graphs. similar to how the global parameters are mate. I changed it to static because they were capitalised and I thought it would have been the writers intention to have this constant and static.

Reply:
You might be right here. I tried to open 2 graphs and I couldn't break it. Here is a profile of the objects showing there are 2 `VisualManager` objects because there are 2 graphs open.

I am still worried this has introduced a threading issue which might not become apparent until there are a lot of graphs open perhaps. It might be too quick and deal with the operations fine just with 2 graphs.

![image](https://user-images.githubusercontent.com/39325530/142301483-9f979186-ee47-470d-8772-0859caa0c7e9.png)